### PR TITLE
PYTHON-3933 Clean up EVG output

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: .evergreen/config.yml [hunk 1]
```diff
@@ -1045,7 +1027,10 @@ functions:
       params:
         script: |
           # Remove all Docker images
-          docker rmi -f $(docker images -a -q) &> /dev/null || true
+          DOCKER=$(command -v docker) || true
```
Comment (Jibola) [inline .evergreen/config.yml:1030]: Trying to get an understanding of `command -v docker`. Does this check if the `docker` `PATH` has been defined or that the command exists in the shell?

Overall, a cool `bash` technique I've just learned! #TIL

**Review by Jibola — APPROVED** [review]
LGTM!

File: .evergreen/config.yml [hunk 2]
```diff
@@ -291,7 +290,10 @@ functions:
             export SKIP_CRYPT_SHARED=1
           fi
 
+          set +x
           ${PREPARE_SHELL}
+          set -o xtrace
```
[the remark this answers is not in the document]
Comment (blink1073) [reply .evergreen/config.yml:295]: I think at one point PREPARE_SHELL itself enabled xtrace, removed.

File: .evergreen/config.yml [hunk 3]
```diff
@@ -291,7 +290,10 @@ functions:
...
+
```
[the remark this answers is not in the document]
Comment (blink1073) [reply .evergreen/config.yml:296]: Refactored this per your next comment.

File: .evergreen/config.yml [hunk 4]
```diff
@@ -923,7 +921,7 @@ functions:
     - command: shell.exec
       params:
         script: |
-          set -o xtrace
+          set +x
           ${PREPARE_SHELL}
```
[the remark this answers is not in the document]
Comment (blink1073) [reply .evergreen/config.yml:925]: I think we originally did that, but it could be a foot-gun

File: .evergreen/config.yml (hunk 1 above)
Comment (blink1073) [reply .evergreen/config.yml:1030]: It searches for an executable of that name on your PATH.

**Pull request merged**